bank system.sql

```diff
@@ -1 +1,402 @@
+/* database creation */
+CREATE DATABASE bank_system;
+GO
 
+USE bank_system;
+GO
+
+CREATE TABLE cities(
+    id INT PRIMARY KEY IDENTITY,
+    city_name VARCHAR(30) NOT NULL
+);
+GO
+
+CREATE TABLE banks(
+    id INT PRIMARY KEY IDENTITY,
+    bank_name VARCHAR(30) NOT NULL
+);
+GO
+
+CREATE TABLE bank_branches(
+    id INT PRIMARY KEY IDENTITY,
+    bank_id INT REFERENCES banks ON DELETE CASCADE NOT NULL,
+    city_id INT REFERENCES cities ON DELETE CASCADE NOT NULL,
+);
+GO
+
+CREATE TABLE social_statuses(
+    id INT PRIMARY KEY IDENTITY,
+    status_name VARCHAR(30) NOT NULL
+);
+GO
+
+CREATE TABLE clients(
+    id INT PRIMARY KEY IDENTITY,
+    status_id INT REFERENCES social_statuses ON DELETE CASCADE NOT NULL,
+    full_name VARCHAR(30) NOT NULL
+);
+GO
+
+CREATE TABLE accounts(
+    id INT PRIMARY KEY IDENTITY,
+    bank_id INT REFERENCES banks ON DELETE CASCADE NOT NULL,
+    client_id INT REFERENCES clients ON DELETE CASCADE NOT NULL,
+    balance MONEY NOT NULL
+    CONSTRAINT unique_accounts UNIQUE (bank_id, client_id)
+);
+GO
+
+CREATE TABLE bank_cards(
+    id INT PRIMARY KEY IDENTITY,
+    account_id INT REFERENCES accounts ON DELETE CASCADE NOT NULL,
+    balance MONEY NOT NULL
+);
+GO
+
+/* database filling */
+INSERT INTO cities (city_name)
+VALUES ('Минск'),
+       ('Витебск'),
+       ('Гродно'),
+       ('Гомель'),
+       ('Могилев'),
+       ('Брест');
+GO
+
+INSERT INTO banks (bank_name)
+VALUES ('Беларусбанк'),
+       ('Белинвестбанк'),
+       ('Паритетбанк'),
+       ('Сбер Банк'),
+       ('МТБанк');
+GO
+
+INSERT INTO bank_branches (bank_id, city_id)
+VALUES (1, 1),
+       (2, 1),
+       (2, 2),
+       (3, 2),
+       (4, 3),
+       (5, 5);
+GO
+
+INSERT INTO social_statuses (status_name)
+VALUES ('Пенсионер'),
+       ('Инвалид'),
+       ('Школьник'),
+       ('Студент'),
+       ('Рабочий');
+GO
+
+INSERT INTO clients (status_id, full_name)
+VALUES (5, 'Мельников Аполлон Владимирович'),
+       (1, 'Ковалёв Максим Федорович'),
+       (5, 'Доронин Леонид Васильевич'),
+       (2, 'Карпов Артур Андреевич'),
+       (4, 'Муравьёв Александр Авдеевич');
+GO
+
+INSERT INTO accounts (bank_id, client_id, balance)
+VALUES (1, 1, 270),
+       (3, 2, 89),
+       (2, 1, 152),
+       (4, 5, 27),
+       (1, 3, 368);
+GO
+
+INSERT INTO bank_cards (account_id, balance)
+VALUES (1, 130),
+       (2, 89),
+       (1, 43),
+       (3, 97),
+       (4, 27);
+GO
+
+/* task 2
+Покажи мне список банков у которых есть филиалы в городе X (выбери один из городов)
+*/
+
+SELECT bank_name
+FROM banks
+    JOIN bank_branches ON bank_id = banks.id
+    JOIN cities ON city_id = cities.id
+WHERE city_name = 'Минск';
+GO
```

Comment:
Check encoding what you use for creation this database or use english words
Right now I get this response
![image](https://user-images.githubusercontent.com/33979910/201544836-fffa60a0-991e-4df6-83ae-6794bd7c5c6f.png)